example/shared/src/commonMain/kotlin/HudPresenter.kt

```diff
@@ -26,16 +28,21 @@ class HudPresenter(private val builder: HUD.Builder) {
     fun showSystem() {
         // SYSTEM style by default
         // No title by default
-        builder.build().present().dismissAfter(3_000)
+
+        MainScope().launch {
```

Comment:
You should probably use MultiplatformMainScope here.

Reply:
@thoutbeckers Thanks! It fixes an issue on iOS but instrumented tests error remains